.github/workflows/ALL-testCoverage-qltycloud.yml

```
@@ -7,7 +7,7 @@ on: [push, workflow_dispatch]
 
 jobs:
   run-coverage:
-    name: "Drupal 11.1.x-dev - PHP 8.3 - PostgreSQL 16"
+    name: "Drupal 11.2.x-dev - PHP 8.3 - PostgreSQL 16"
```

We ended up with a version mismatch here when we realized we needed to keep the test coverage at 11.1 🙈 

```suggestion
    name: "Drupal 11.1.x-dev - PHP 8.3 - PostgreSQL 16"
```